common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -154,6 +164,15 @@ def student_view(self, context):
 
         fragment = Fragment()
 
+        if self.is_time_limited:
+            view_html = self._time_limited_student_view(context)
+
+            # Do we have an alternate rendering
+            # from the edx_proctoring subsystem?
+            if view_html:
```

Comment:
When do we have an alternate rendering? Do we always expect one if self.is_time_limited is True?